front/services/address/GooglePlaceAutocomplete.js

```diff
@@ -1,12 +1,10 @@
-export default class GooglePlaceAutocomplete {
-    constructor(wrapper, addressField, cityField, zipCodeField, countryField, inputClassNames = '') {
-        this._wrapper = wrapper;
-
-        this._addressField = addressField;
-        this._cityField = cityField;
-        this._zipCodeField = zipCodeField;
-        this._countryField = countryField;
+import EventEmitter from 'events';
 
+export default class GooglePlaceAutocomplete extends EventEmitter {
```

Comment:
Don't know anything about JS but it sounds weird in terms of OOP to extends from EventEmitter.
Can't we use composition or an EventDispatcher.
No need to bother too much with that of course.

Reply:
why it's weird? 
in JS you can already do `object.addEventListener()` where the object is DOM element.

Reply:
In terms of OOP, it sounds weird ;-)